spec/mailers/previews/notification_mailer_preview.rb

```diff
@@ -0,0 +1,10 @@
+# Preview all emails at http://localhost:3000/rails/mailers/notification_mailer
+class NotificationMailerPreview < ActionMailer::Preview
+  def send_notifications
+    NotificationMailer.send_notifications(user, 
+    											task, 
+    											task_category, 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.<br>Trailing whitespace detected.